### DISCUSSION TITLE
Is there any way to bind real kv when using `wrangler pages dev` command?

### DISCUSSION THREAD

**Discussion by hyunjunian** [cloudflare/workers-sdk#2492] [Q&A]
I use `wrangler pages dev` with `-b KV_NAMESPACE`. But it only creates KV_NAMESPACE locally.
I want to use real KV_NAMESPACE which I created in cloudflare console.
Is there any way to do this?

**Comment by JacobMGEvans** [comment]
Were you able to resolve this issue?

**Reply by hyunjunian** [reply]
Does the latest version of wrangler support this feature?

**Reply by JacobMGEvans** [reply]
No, there is no current feature that would allow for this.
